api/app/src/main/kotlin/packit/model/PacketIdCountsDTO.kt

```diff
@@ -0,0 +1,13 @@
+package packit.model
+
+// Projection class for PacketRepository.findIdCountDataByName
+interface IPacketIdCountsDTO
```

Comment:
also im running all my tests locallly and making sure they pass.. maybe we can just rely on build actions running on the main feature branch this will be merged into